src/cli/options.rs

```diff
@@ -74,3 +69,44 @@ fn get_many_from_args<'a, T: Clone + Send + Sync + 'static>(
 ) -> Vec<&'a T> {
     args.get_many::<T>(name).unwrap_or_default().collect()
 }
+
+fn get_skip_lints(args: &ArgMatches) -> Vec<LintKind> {
+    let mut skip: Vec<LintKind> = get_many_from_args::<LintKind>(args, "skip")
+        .into_iter()
+        .map(|l| l.to_owned())
+        .collect();
+
+    get_skip_lints_from_env_vars(&mut skip);
+    skip
+}
+
+fn get_skip_lints_from_env_vars(skip: &mut Vec<LintKind>) {
+    let env_skip_vars = env::var("DOTENV_LINTER_SKIP");
```

Comment:
You don't need to do it manually, `clap` has embedded support
https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.env

Reply:
I've replaced my logic with clap env